src/test/java/Homework21.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest{
+    String newPlaylistName = "Homework21 Playlist";
+    @Test
+    public void renamePlaylist() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
For some reason, i wasn't able to pass the test without adding the thread.sleep but it looks like it worked now so i deleted them!